app/containers/EmojiPicker/EmojiCategory.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { FlatList } from 'react-native-gesture-handler';
+import { FlatList } from 'react-native';
```

Comment:
We could just make RNGH work https://docs.swmansion.com/react-native-gesture-handler/docs/fundamentals/installation#android
It shouldn't be an issue though.